src/main/java/io/quarkus/bot/release/ReleaseInformation.java

```diff
@@ -102,20 +132,33 @@ public boolean isDot0() {
         return Versions.isDot0(version);
     }
 
+    /**
+     * @return whether this version is the CR1
+     */
     @JsonIgnore
     public boolean isFirstCR() {
         return "CR1".equalsIgnoreCase(qualifier);
     }
 
+    /**
+     * @return whether this is a major new release (e.g. 4.0.0.CR1)
+     */
     public boolean isMajor() {
         return major;
     }
 
+    /**
+     * @return whether the origin branch is the main branch (which is the default, should only be false when release the CR1 of a new LTS)
```

Comment:
Hold on, but when releasing an LTS maintenance release, it will also be *not* from main, no?

Reply:
I don't see any new changes here (apart from splitting the sentence into two lines)

Reply:
I think I will just point to the other method Javadoc to avoid duplication.

Reply:
Pushed something. Hopefully, it's good enough now. Otherwise, maybe we could have a quick call instead, might be faster :).